qbraid/runtime/azure/result_builder.py

```
@@ -204,7 +204,9 @@ def _format_quantinuum_results(self) -> dict[str, Any]:
             if classical_register != "access_token"
         ]
         counts = {}
-        combined_bitstrings = ["".join(bitstrings) for bitstrings in zip(*all_bitstrings)]
+        # Quantinuum reports c[0] as the rightmost bit; qBraid puts it first. Confirmed
+        # against production jobs whose ideal outcome distinguishes the two orders.
+        combined_bitstrings = ["".join(bitstrings)[::-1] for bitstrings in zip(*all_bitstrings)]
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)